### PR TITLE
Added the pipeline

### PR DESCRIPTION
**Pull request by nvnieuwk**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/CenterForMedicalGeneticsGhent/nf-cmgg-wisecondorx/tree/master/.github/CONTRIBUTING.md)
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: workflows/cmgg-wisecondorx.nf [hunk 1]
```diff
@@ -48,7 +46,11 @@ include { INPUT_CHECK } from '../subworkflows/local/input_check'
 //
 // MODULE: Installed directly from nf-core/modules
 //
-include { FASTQC                      } from '../modules/nf-core/fastqc/main'
+
+include { SAMTOOLS_FAIDX              } from '../modules/nf-core/samtools/faidx/main'
+include { SAMTOOLS_INDEX              } from '../modules/nf-core/samtools/index/main'
+include { WISECONDORX_CONVERT         } from '../modules/nf-core/wisecondorx/convert/main'
+include { WISECONDORX_NEWREF          } from '../modules/nf-core/wisecondorx/newref/main'
 include { MULTIQC                     } from '../modules/nf-core/multiqc/main'
```
Comment (matthdsm) [inline workflows/cmgg-wisecondorx.nf:54]: also drop MQC?

**Review by matthdsm — APPROVED** [review]

File: workflows/cmgg-wisecondorx.nf (hunk 1 above)
Comment (nvnieuwk) [reply workflows/cmgg-wisecondorx.nf:54]: It's only used to show a summary of the versions used. I can remove it if that's not necessary

**Comment by nvnieuwk** [comment]
TODO: fix CI not working

**Comment by nvnieuwk** [comment]
HPC test ran succesfully!

File: workflows/cmgg-wisecondorx.nf (hunk 1 above)
Comment (nvnieuwk) [reply workflows/cmgg-wisecondorx.nf:54]: It contains useful information (like the command used, the tool versions and the parameters used for the run). I think this is useful to save for reproducibility reasons. (And it barely adds anything to the runtime or the disk space of the output folder)

**Pull request merged**